daemon/mgr/container.go

```diff
@@ -941,6 +941,19 @@ func (mgr *ContainerManager) Update(ctx context.Context, name string, config *ty
 		}
 	}()
 
+	if config.Hostname != "" {
+		if c.HostConfig.NetworkMode == "host" {
+			return fmt.Errorf("host container %s cannot update hostname", c.ID)
+		}
+
+		c.Config.Hostname = strfmt.Hostname(config.Hostname)
+		if c.IsRunning() {
+			if err = updateHostnameForRunningContainer(config.Hostname, c); err != nil {
+				return err
```

Comment:
How about adding a log  to record when updating the hostname of container successful?

Reply:
Have added a log to record it .